modules/users/client/views/authentication/signup.client.view.html

```diff
@@ -26,6 +26,17 @@ <h3 class="col-xs-12 text-center">Sign up</h3>
           </div>
         </div>
         <div class="form-group" show-errors>
+          <label for="roles">Role</label>
```

Comment:
Scratch this, we'll just do it from admin/addnewuser and admin/approval

Reply:
Would it be convenient for the admin to see what role the user is applying for? I believe that the admin would have a hard time to search up the role for every user that is applying to the system. I feel like adding this feature would make life easier for the admin.

Reply:
I don't really think it's a concern worth spending time on, and my real reason is that I had a good deal of trouble with harnessing "default" roles on the approval view so the work would go to waste anyway. It's not part of a story and we don't need the functionality.